### DISCUSSION TITLE
How can i improve flask documentation ?

### DISCUSSION THREAD

**Discussion by MrinmoyHaloi** [pallets/flask#4195] [Ideas]
I looked flask documentation today. It doesn't looks good for me. I want to improve it help other learn flask faster and easily. I wanted to add images to show the output. I think it will be very helpful.

**Comment by ThiefMaster** [comment]
Output for what exactly? I don't see how adding screenshots would be useful in our documentation...

**Reply by MrinmoyHaloi** [reply]
a screenshot explain what did the code do. 
And how can I improve other things of the documentation

**Reply by ThiefMaster** [reply]
https://flask.palletsprojects.com/en/2.0.x/tutorial/

There are some screenshots. I don't understand what else you would screenshot there...

**Reply by MrinmoyHaloi** [reply]
ok. but how can i improve the visuals of documentation for the website.

**Reply by ThiefMaster** [reply]
Once again, why do you think it needs to have "improved visuals"? Make an explicit example of what you would change.

**Reply by MrinmoyHaloi** [reply]
i want flask docs to be good as django's

**Reply by ThiefMaster** [reply]
again, make an example. on a quick glance i can't see how our docs are worse than theirs

**Reply by MrinmoyHaloi** [reply]
i don't say that its worse. but doesn't looks good as theirs. The colors of django's website is better

**Reply by ThiefMaster** [reply]
that's completely subjective...

**Reply by MrinmoyHaloi** [reply]
so what do you think ?

**Reply by ThiefMaster** [reply]
I think both look fine.
